packages/editor/src/components/post-publish-panel/style.scss

```diff
@@ -85,18 +85,21 @@
 .post-publish-panel__postpublish-buttons {
 	display: flex;
 	align-content: space-between;
+	flex-wrap: wrap;
+	margin: -5px;
```

Comment:
Where is `5px` coming from? We have a bunch of `$grid-size` variables and they're `4px`-based I think.

Reply:
The original margin between the two buttons was 10px. Since we need to allow for either a right-margin or bottom-margin on the first button, depending on how they use the space, I've halfed the original margin, applied it to both buttons equally, then applied a negative margin to the whole container so that they end up in the same place.

Basically, this creates a flexible gutter between items instead of a margin, and then adjusts to make sure everything still lines up properly. It's ... not not a hack, but it works.